From 78c8a914ab63b2b8bd680d4d9695311ab0208099 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 18 Feb 2024 20:03:05 +0800
Subject: [PATCH] misc: use pkexec in .desktop entry

The current implementation relies on KDE su, which is no longer supported.
---
 misc/grub-customizer.desktop | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/misc/grub-customizer.desktop b/misc/grub-customizer.desktop
index 7e405e7..aac8041 100755
--- a/misc/grub-customizer.desktop
+++ b/misc/grub-customizer.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Grub Customizer
 Comment=Customize the bootloader (GRUB2 or BURG)
-Exec=grub-customizer
+Exec=pkexec grub-customizer
 Terminal=false
 X-MultipleArgs=false
 Type=Application
@@ -9,5 +9,4 @@ StartupNotify=false
 Categories=System;Settings;
 Icon=grub-customizer
 X-Ubuntu-Gettext-Domain=grub-customizer
-X-KDE-SubstituteUID=true
 Keywords=GRUB;GRUB2;BURG;configuration;settings;menu;
-- 
2.43.0

